dhis-2/dhis-api/src/main/java/org/hisp/dhis/resourcetable/ResourceTable.java

```diff
@@ -37,6 +37,10 @@
  */
 public abstract class ResourceTable<T>
 {
+    public static final int OLDEST_YEAR_PERIOD_SUPPORTED = 1975;
+
+    public static final int NEWEST_YEAR_PERIOD_SUPPORTED = 2024;
```

Comment:
hmm, what about to use (current year) + 2

Reply:
Done